vendor/foxy/cmake/certify.cmake

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.11)
+
+include(FetchContent)
+
+if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.24")
+    # Affects robustness of timestamp checking on FetchContent dependencies.
+    cmake_policy(SET CMP0135 NEW)
+endif ()
+
+
```

Comment:
TODO: This needs to be updated to the LaunchDarkly [fork](https://github.com/launchdarkly/certify/), but there are some pulls that need to be merged before it's usable.